test/helpers/default-functions.ts

```diff
@@ -70,52 +84,28 @@ export function defaultFunctions(): void {
 	);
 
 	/**
-	 * Get installed extensions on wiki
+	 * Skip test if extension is not installed (present) on the Wikibase server
 	 */
 	browser.addCommand(
-		'getInstalledExtensions',
-		async ( server: string ): Promise<string[] | undefined> => {
-			const result = await browser.makeRequest(
-				`${server}/w/api.php?action=query&meta=siteinfo&siprop=extensions&format=json`
+		'skipIfExtensionNotPresent',
+		async (
+			test: Context,
+			extension: string
+		): Promise<void> => {
+			const installedExtensions = await getInstalledExtensions(
+				testEnv.vars.WIKIBASE_URL
```

Comment:
At the risk of being annoying
The switch from `getInstalledExtensions` to `skipIfExtensionNotPresent` does seem like a discrete, separable change
Thoughts on splitting that out?

Reply:
This is actually required by this changeset. `skipIfExtensionNotPresent` relies upon a testEnv.var which is now, by design, not available until the `testEnv` is booted (previously it was relying on an envvar which was there from the start). Previously `getInstalledExrwnsions` was on browser, and `skipIfExtensionNotPresent` was not. One of them always had to be not in `defaultFunctions` so it could reference the browser object with the other method already present on it. I've switched which one is on browser here as `akipIfExtensionNotPresent` also needs access to the `testEnv` which is within scope in `defaultFunctions`. Hope that makes sense. It is confusing, though the code here is I think no less clear than what was there before, and this change is necessary in this branch.